services/blockchain-indexer/shared/utils/accountUtils.js

```diff
@@ -40,6 +40,8 @@ const getAccountsTable = () => getTableInstance(
 
 const getIndexedAccountInfo = async (params, columns) => {
 	if (!('publicKey' in params) || params.publicKey) {
+		params.limit = 1;
```

Comment:
What is the reason to define `limit` here?

Reply:
we only return the first row. Added to optimize the query incase we forget to add the param while calling.
https://github.com/LiskHQ/lisk-service/blob/f75c1971da84b39ec439815522c0823c30b71f47/services/blockchain-indexer/shared/utils/accountUtils.js#L46